frontend/src/components/store/PluginCard.tsx

```diff
@@ -113,9 +113,20 @@ const PluginCard: FC<PluginCardProps> = ({ plugin }) => {
             }}
             className="deckyStoreCardInfo"
           >
-            <p className={joinClassNames(staticClasses.PanelSectionRow)}>
+            <p className={joinClassNames(staticClasses.PanelSectionRow)} style={{ marginTop: '0px' }}>
```

Comment:
```suggestion
            <p className={joinClassNames(staticClasses.PanelSectionRow)} style={{ marginTop: '0px' marginLeft: '16px'}}>
```
not sure how prettier will feel about this one.